openedx/frontend-app-ora-grading
# fix(deps): update dependency react-pdf to v7 [security] #1813

### Description
Updates react-pdf to version 7 and improves PDF worker configuration by using CDN delivery.

### Changes
- Update PDF.js worker configuration to use CDN
- Remove local PDF worker file from public directory
- Add TextLayer CSS import for better PDF rendering
- Configure worker source URL to use CDNJS
- Improve PDF rendering stability and security

### Testing Instructions
1. Navigate to any page with PDF preview functionality
2. Verify PDF files load and display correctly
3. Test PDF navigation (next/previous pages)
4. Verify page number input works
5. Check PDF rendering in different screen sizes

### Related Information
- Issue: #1813
- Previous worker configuration used local file
- New configuration uses CDNJS for better performance and security
